pipelines/operator-hosted-pipeline.yml

```diff
@@ -0,0 +1,255 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: operator-hosted-pipeline
+spec:
+  params:
+    - name: git_pr_branch
+    - name: git_pr_title
+    - name: git_pr_url
+    - name: git_repo_url
+    - name: git_username
+    - name: pyxis_url
+    - name: bundle_path
+    - name: preflight_latest_version
+    - name: ci_latest_version
+    - name: internal_registry_url
+      default: default-route-openshift-image-registry.apps-crc.testing
+    - name: quay_url
+      default: quay.io
```

Comment:
Can these be combined into a single param (e.g. `registry_url`)? We shouldn't be restricted to an internal registry and we can push everything to Quay.